### DISCUSSION TITLE
Discussions Enabled

### DISCUSSION THREAD

**Discussion by therealmistericraft** [therealmistericraft/MisteriBot#53] [General]
## 👋 Hi everyone,
I set up discussions. I still do not know what exactly they are for, but it cannot do anything wrong. One usecase I could imagine would be discussing about new features, their implementation and exchange of thoughts.

But ***please*** be friedly to each other.
Have fun,
*Jan // MisteriCraft*

**Comment by ghost** [comment]
Hello 👋

**Reply by DanilTap** [reply]
hi
